### ISSUE TITLE
Run without Dock icon

### ISSUE THREAD

**Issue by JayBrown** [tylerhall/roar#2]
Thank you for the app… pretty great. Any chance to make it run as a background process without a Dock icon?

**Comment by tylerhall** [comment]
Absolutely, you could. Right now, the app was built as quickly as possible. Removing the Dock icon would also remove the menu bar, which I think will break the 1-4 action shortcut keys. That action window would need to be rebuilt (correctly, this time) to handle this.
